packages/server/src/middleware/nuxt.js

```diff
@@ -12,6 +12,10 @@ export default ({ options, nuxt, renderRoute, resources }) => async function nux
   res.statusCode = 200
   try {
     const result = await renderRoute(url, context)
+
+    // If result === false, rendered a loading screen
+    if (!result) return
```

Comment:
```suggestion
    if (!result) {
      return
    }
```

No inline-ifs! :stuck_out_tongue_winking_eye:

Reply:
Done